packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed issue where numeric `encoding` value caused `DownloadJobs.downloadSpoolContentCommon` method to fail. [#2472](https://github.com/zowe/zowe-cli/pull/2472)
```

Comment:
Let's add the missing articles: Fixed **an** issue where **a/the** numeric `encoding` value caused **the** `DownloadJobs.downloadSpoolContentCommon` method to fail